.gitignore

```diff
@@ -127,3 +127,12 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Upstream git repo
+upstream/
+libinjection/libinjection.h
```

Comment:
Maybe `*.pyc` also?

Reply:
For sure. That is already in the `.gitignore` file [here](https://github.com/libinjection/python3-libinjection/blob/84b39d598056a01a0860c659dcfc33beeb79fff5/.gitignore#L3). We are only seeing the addition/changes here. :)